.goreleaser.yml

```diff
@@ -1,16 +1,14 @@
-before:
-  hooks:
-    - go mod download
-
 builds:
-  - env:
-      - CGO_ENABLED=0
-    goos:
+  - goos:
       - linux
       - darwin
     goarch:
       - amd64
       - arm64
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s -w -X main.VERSION={{ .Version }}
```

Comment:
Potential for adding other flags here if needed.

Reply:
We usually had version, sha, and date compiled passed onto it and expose all of that in the /status endpoint, or even the /health endpoint

Reply:
ya let me add that, thats the default for GoReleaser too :)